docker-entrypoint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e
+
+if [ "$1" = 'apache2-foreground' ]; then
+    wait-for-it $DB_HOST:${DB_PORT:=3306}
+    cd /var/www/html
+    runuser -u www-data -- php artisan migrate
```

Comment:
```suggestion
    runuser -u www-data -- php artisan migrate --force
```

Isn't this required?